samples/bluetooth/path_loss_monitoring/README.rst

```diff
@@ -0,0 +1,138 @@
+.. _bluetooth_path_loss_monitoring:
+
+Bluetooth: Path loss monitoring
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Path Loss Monitoring sample demonstrates how to evaluate Bluetooth® LE signal quality using the Path Loss Monitoring feature.
+
+It consists of a Central and a Peripheral device.
+
+The Central continuously monitors the signal strength and classifies the connection quality using onboard LEDs.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+You must use two compatible development kits to run this sample: one programmed with the Central application, and one with the Peripheral application.
+
+Overview
+********
+
+This sample illustrates Bluetooth® LE Path Loss Monitoring by calculating the signal degradation between two connected devices.
+
+The Central application scans for a Peripheral named ``path_loss_peripheral`` and connects to it.
+
+After a successful connection, the Central measures path loss using the formula:
+
+::
+
+   path loss = Tx Power - RSSI
+
+The application defines three signal quality zones based on the path loss value:
+
+- **Low path loss (strong signal):** LED0 is turned on.
+- **Medium path loss:** LED1 is turned on.
+- **High path loss (weak signal):** LED2 is turned on.
+
+The Peripheral simply advertises its presence and accepts connections.
+
+User interface
+**************
+
+Central:
+
+* **LED0:** Indicates **Low** path loss (good signal).
+* **LED1:** Indicates **Medium** path loss.
+* **LED2:** Indicates **High** path loss (weak signal).
+
+No buttons are used.
+
+Peripheral:
+
+No direct user interface.
+
+The device advertises under the name ``path_loss_peripheral``.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/path_loss_monitoring`
+.. include:: /includes/build_and_run.txt
+
+.. |sample_or_app| replace:: sample
+.. |ipc_radio_dir| replace:: :file:`sysbuild/ipc_radio`
+.. include:: /includes/ipc_radio_conf.txt
+
+Testing
+=======
+
+To test the sample:
+
+1. Program one DK with the **Peripheral Path Loss Monitoring** sample.
+#. Program another DK with the **Central Path Loss Monitoring** sample.
```

Comment:
Use links instead of the names in bold.
But wait, are there separate samples for Peripheral and Central?